docs/technical_reference/unit_models/reverse_osmosis_0D.rst

```diff
@@ -138,26 +141,27 @@ Equations
 .. csv-table::
    :header: "Description", "Equation"
 
-   "Water flux across membrane", ":math:`J_{w, x} = \rho_w A(P_{f,x} - P_p - (\pi_{f,m,x}-\pi_{p,x}))`"
-   "Solute flux across membrane", ":math:`J_{s, x} = B(C_{f,m,x} - C_p)`"
-   "Average water flux across membrane", ":math:`J_{w, avg} = \frac{1}{2}\sum_{x} J_{w, x}`"
-   "Average solute flux across membrane", ":math:`J_{s, avg} = \frac{1}{2}\sum_{x} J_{s, x}`"
-   "Permeate mass flow by component j", ":math:`M_{p, j} = A_m J_{j,avg}`"
-   "Membrane-interface concentration", ":math:`C_{f,m}=CP_{mod}*C_f=C_f\exp(\frac{J_w}{k_f})-\frac{J_s}{J_w}(\exp(\frac{J_w}{k_f})-1)`"
-   "Concentration polarization modulus",":math:`C_{f,m}/C_f`"
-   "Mass transfer coefficient",":math:`k_f=\frac{D Sh}{d_h}`"
-   "Sherwood number",":math:`Sh=0.46 (Re Sc)^{0.36}`"
-   "Schmidt number",":math:`Sc=\frac{\mu}{\rho D}`"
-   "Reynolds number",":math:`Re=\frac{\rho v_f d_h}{\mu}`"
-   "Hydraulic diameter",":math:`d_h=\frac{4\epsilon_{sp}}{2/h_{ch} + (1-\epsilon_{sp})8/h_{ch}}`"
-   "Cross-sectional area",":math:`A_c=h_{ch}W\epsilon_{sp}`"
-   "Membrane area",":math:`A_m=LW`"
-   "Pressure drop",":math:`ΔP=(\frac{ΔP}{Δx})_{avg}L`"
-   "Feed-channel velocity",":math:`v_f=Q_f/A_c`"
-   "Friction factor",":math:`f=0.42+\frac{189.3}{Re}`"
-   "Pressure drop per unit length",":math:`\frac{ΔP}{Δx}=\frac{1}{2d_h}f\rho v_f^{2}`"
-   "Component recovery rate",":math:`R_j=\frac{M_{p,j}}{M_{f,in,j}}`"
-   "Volumetric recovery rate",":math:`R=\frac{Q_{p}}{Q_{f,in}}`"
+   "Solvent flux across membrane", ":math:`J_{solvent} = \rho_j A(P_{f} - P_p - (\pi_{f}-\pi_{p}))`"
```

Comment:
Since `j` in `\rho_j` only applies to when `j` is a solvent, you should remove `j` and replace with `solvent` or have no subscript (even though one could assume `j` only applies to solvents because the property is labeled "Solvent flux")

Reply:
Also, we should replace dens_solvent in the vars table to be described as "Solvent density" rather than "Mass density of pure water". Furthermore, dens_solvent is currently a parameter and not a variable.

Reply:
Oops yes